virtual/vagrant_startup.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+. /etc/os-release
+
+# Install Software
+case "$ID_LIKE" in
+    rhel*)
+        # Install Vagrant
+	set -xe
+
+
+	# update yum
+	sudo yum  update
+
+	# install essential packages and tools
+	sudo yum -y install wget
+	sudo yum group install -y "Development Tools"
+	sudo yum install -y centos-release-qemu-ev qem-kvm-ev qemu-kvm libvirt virt-install bridge-utils libvirt-devel  libxslt-devel libxml2-devel libvirt-devel libguestfs-tools-c
+
+	# set up networking for Vagrant VMs
+	#sudo echo "net.ipv4.ip_forward = 1"|sudo tee /etc/sysctl.d/99-ipforward.conf
+	#sudo sysctl -p /etc/sysctl.d/99-ipforward.conf
+
+	# start up libvirt as our VM method for Vagrant
+	sudo usermod -a -G libvirt $(whoami)
+	sudo systemctl enable libvirtd
+	sudo systemctl start libvirtd
+
+
+	# install vagrant (frozen at 2.2.3 to avoid various issues)
+	pushd "$(mktemp -d)"
+	wget https://releases.hashicorp.com/vagrant/2.2.3/vagrant_2.2.3_x86_64.rpm -O vagrant.rpm
+	#sudo rpm -i vagrant.rpm
+	sudo yum -y localinstall vagrant.rpm
+	popd
+
+	# install other dependencies
+	sudo yum install -y sshpass
+
+	# install kvm packages
+	sudo yum install -y qemu-kvm libvirt-bin libvirt-dev bridge-utils libguestfs-tools
+	sudo yum install -y qemu virt-manager firewalld OVMF
+
+	# install vagrant plugins
+	vagrant plugin install vagrant-hostmanager vagrant-libvirt
+	vagrant plugin install vagrant-host-shell vagrant-scp vagrant-mutate
+
+	#set up Vagrantfile and start up the configuration in Vagrant
+	export DEEPOPS_VAGRANT_FILE="/home/$(whoami)/deepops/virtual/Vagrantfile-centos"
+
+
+
+	# End Install Vagrant
+        ;;
+    debian*)
+        # Install Vagrant
+
+set -xe
+
+	# update apt
+	sudo apt update
+
+	# install build-essential tools
+	sudo apt install build-essential
+
+	# install vagrant (frozen at 2.2.3 to avoid various issues)
+	pushd "$(mktemp -d)"
+	wget https://releases.hashicorp.com/vagrant/2.2.3/vagrant_2.2.3_x86_64.deb -O vagrant.deb
+	sudo dpkg -i vagrant.deb
+	popd
+
+	# install other dependencies
+	sudo apt install -y sshpass
+	
+	# install kvm packages
+	sudo apt install -y qemu-kvm libvirt-bin libvirt-dev bridge-utils libguestfs-tools
+	sudo apt install -y qemu ovmf virt-manager firewalld
+
+	# install vagrant plugins
+	vagrant plugin install vagrant-hostmanager vagrant-libvirt
+	vagrant plugin install vagrant-host-shell vagrant-scp vagrant-mutate
+
+	#set up Vagrantfile and start up the configuration in Vagrant
+	export DEEPOPS_VAGRANT_FILE="/home/$(whoami)/deepops/virtual/Vagrantfile-ubuntu"
+
+
+	# End Install Vagrant
+        ;;
+    *)
+        echo "Unsupported Operating System $ID_LIKE"
+        echo "Please install Ansible, Git, and python-netaddr manually"
```

Comment:
Nit: Should mention Vagrant here as well, since we'll have skipped those install steps.

Reply:
good catch.. This was unchanged remnants of the scripts/setup.sh file that I copied. Completely agree